html_css/intermediate_html/form-validations.md

```diff
@@ -4,7 +4,7 @@ Validations allow us to set specific constraints or rules that determine what da
 
 Validations are a vital ingredient in well-designed forms. They help protect our backend systems from receiving incorrect data, and they help make the experience of interacting with our form as dead-stupid-simple as possible for our users.
 
-This lesson will explore some of the built-in validations you can use with HTML forms. We will also dive into creating custom validations using JavaScript.
+This lesson will explore some of the built-in validations you can use with HTML forms.
```

Comment:
This should include mention that the user will also learn about styling for validations.